util/sample_collections.py

```diff
@@ -188,15 +235,18 @@ def read_row(self, row_index, *columns):
                 column_data[columns.index(index)] = self.sdb_file.read(chunk_len)
                 found += 1
                 if found == len(columns):
-                    return tuple(column_data)
+                    return tuple(column_data) if len(column_data) > 1 else column_data[0]
```

Comment:
Instead of returning different types here for convenience in the callers, you can always return a tuple and let callers use destructuring assignment.

Reply:
Agree - will fix it